test-utils/state-viewer/src/rocksdb_stats.rs

```diff
@@ -101,7 +105,13 @@ fn main() {
     }
 
     let mut column_data_list: Vec<&Data> = column_data.values().collect();
-    column_data_list.sort_by_key(|data| u64::MAX - data.estimated_table_size);
-    eprintln!("Printing stats ...");
-    println!("{}", serde_json::to_string(&column_data_list).unwrap());
+    column_data_list.sort_by_key(|data| std::cmp::Reverse(data.estimated_table_size));
+    let result = serde_json::to_string_pretty(&column_data_list).unwrap();
+
+    eprintln!("Dumping stats ...");
+    match file {
+        None => println!("{}", result),
+        Some(file) => std::fs::write(file, result)?,
+    }
```

Comment:
Rather than passing in an `Option<PathBuf>`, a more general solution is to pass `&mut dyn Write`, that way the code doesn't have to care where does the output go.